docs/ca-signed-workflow.md

```diff
@@ -1,7 +1,7 @@
 # Sign and verify an artifact with a certificate signed by a trusted CA in Azure Key Vault
 > **Note** The following guide can be executed on Linux bash, macOS Zsh and Windows WSL
 1. [Install the Azure CLI](https://learn.microsoft.com/cli/azure/install-azure-cli)
-2. Log in to Azure with Azure CLI, set the subscription and make sure the `GetCertificate` and `Sign` permission have been granted to your role:
+2. Log in to Azure with Azure CLI, set the subscription and make sure the `Secrets Get` and `Key Sign` permission have been granted to your role:
```

Comment:
I think this permission prerequisite is required for Azure Key Vault, right? Let's be more specific here. Is there a related Azure doc that users can follow to configure or check the permission? If so, we need to add the link here.

Reply:
Updated to include the script to set the permission and the link for AKV access policy.